pkg/cmd/config.go

```diff
@@ -45,6 +46,9 @@ const (
 	defaultCloudProviderName = "aws"
 )
 
+//go:embed example-manifest.tmpl.yaml
+var exampleManifest string
```

Comment:
FYI: `kubeone config print -f` fails with:
```
Error: runtime: testing marshal/unmarshal
error unmarshaling JSON: while decoding JSON: json: unknown field "podSecurityPolicy"
```

Reply:
Yeah, after rebasing and getting podSecurityPolicy removed we got the error. fixed now.